linkerd/docs/protocol-http.md

```diff
@@ -165,6 +165,47 @@ Key | Default Value | Description
 dstPrefix | `http` | The `dstPrefix` as set in the routers block.
 urlPath | N/A | A path from the URL whose number of segments is set in the identifier block.
 
+<a name="header-identifier"></a>
+### Header Identifier
+
+kind: `io.l5d.header`
+
+With this identifier, HTTP requests are turned into names based only on the
+value of an HTTP header.  If the header value is a valid path, that path is
+used.  Otherwise, the header value is converted to a path with one path segment.
+
+#### Namer Configuration:
+
+> With this configuration, the value of the `my-header` HTTP header will be used
+as the logical name.
+
+```yaml
+routers:
+- protocol: http
+  identifier:
+    kind: io.l5d.header
+    header: my-header
+  servers:
+  - port: 5000
+```
+
+Key | Default Value | Description
+--- | ------------- | -----------
+header | `l5d-dst-concrete` | The name of the HTTP header to use
```

Comment:
That's not bad.  Now that I look at the [header docs](https://buoyant.io/slate-linkerd/0.7.5/linkerd/#user-headers), is it confusing to reuse the l5d-dst prefix?

> The informational headers linkerd emits on outgoing requests.
> ...
> Applications are not required to forward these headers on downstream requests.

Perhaps `l5d-name` or `l5d-target`?

Names are hard.